### PR TITLE
verification ui

### PR DESCRIPTION
**Pull request by jonpitch**

### CODE REVIEW

**Comment by jonpitch** [comment]
Apologies to whomever does the code review, there's a lot to look at. I'm also not 100% sure I hit the comps to what @jvitelli and Jerrod are looking for. I'll touch base with them as soon as possible.

Some reflections on the changes:

- I made an effort to not use Materialize. Having to initialize components after render causes a lot of issues for us while testing as well as it creates a lot of extra markup.
- I always like to make an effort to use semantic markup (at least to the best I know how). There are a lot of inherited styles for semantic things like `footer`, `main`, `section`, etc. If we had all the time in the world, I'd love to ditch Materialize.
- Liquid fire is really great. It's also a huge pain. Most animations render themselves with `position: absolute` in order to calculate the final DOM height and then animate to that state. This is a huge burden to work around in CSS.
- All of the integration tests now use page object.

cc @kikemoto @zkrzyzanowski

**Comment by jonpitch** [comment]
@jvitelli @kikemoto @zkrzyzanowski UI is now good enough for MVP.

File: addon/components/verification-actionbar.js [hunk 1]
```diff
@@ -38,56 +36,56 @@ export default Component.extend({
...
-  reinitializeCollapsible: observer('expanded', function() {
-    this.enableCollapsibleVerification();
-  }),
+  actions: {
 
-  verificationsChanged: on('init', observer('question.verifications', function() {
-    this.enableCollapsibleVerification();
-  })),
+    // show/hide content
+    toggle(section) {
```
Comment (kikemoto) [inline addon/components/verification-actionbar.js]: @jonpitch Much cleaner!

File: scripts/gh-pages.sh [hunk 2]
```diff
@@ -1,16 +1,24 @@
...
+  # git init
   git config --global user.name "The Impact Bot"
   git config --global user.email "technology@bcorporation.net"
   git config --global push.default simple
 
-  # npm run deploy
-  ember github-pages:commit --message "deploy gh-pages from Travis"
-  git push --force --quiet "https://${GH_ACCESS_TOKEN}@github.com/b-lab-org/ember-cli-impact-core.git" master:gh-pages > /dev/null 2>&1
+  # debug
+  git remove -v
```
Comment (kikemoto) [inline scripts/gh-pages.sh]: @jonpitch Could you explain what this change is doing?

Comment (jonpitch) [inline scripts/gh-pages.sh]: @kikemoto You found a typo, thanks!

**Pull request merged**